app/create_app.py

```diff
@@ -140,9 +141,24 @@ def find_fund_in_request():
             fund = get_fund_data(request.args.get("fund_id"), as_dict=True)
         elif request.args.get("fund"):
             fund = get_fund_data_by_short_name(request.args.get("fund"))
+        elif (
+            request.args.get("application_id")
+            or request.view_args.get("application_id")
+            or request.form["application_id"]
+        ):
+            application_id = (
+                request.args.get("application_id")
+                or request.view_args.get("application_id")
+                or request.form["application_id"]
+            )
```

Comment:
can we simplify this?
```suggestion
        elif application_id := (
            request.args.get("application_id")
            or request.view_args.get("application_id")
            or request.form["application_id"]
        ):
```